izanami-clients/node/index.js

```diff
@@ -315,34 +315,44 @@ module.exports = {
     clientSecret: '--',
     env: 'PROD'
   },
-  featureClient: (config) => new IzanamicFeatureClient(config),
+  featureClient: (config) => new IzanamiFeatureClient(config),
   configClient: (config) => new IzanamiConfigClient(config),
   experimentsClient: (config) => new IzanamiExperimentsClient(config),
   expressProxy(config) {
 
-    const { featureClient, configClient, experimentsClient, app, sessionPath, experimentWonPath, experimentDisplayedPath, path } = config;
+    const {
+      featureClient,
+      configClient,
+      experimentsClient,
+      app,
+      sessionPath = '/api/me',
+      featureContextFromRequest = (req) => ({user: req.user_email}),
```

Comment:
I have some questions about the features and experiment contexts: I have extracted the functions to make the proxy configurable, but I don't see where this `req.user_email` comes from in the initial implementation.

This implies some sort of request body parser, or some auth, and I don't see any mention anywhere.

I am willing to extend the proxy to include these contexts in my work on #227, as the React client relies on the proxy to run

Reply:
Sorry for a late response...

There are several issues : 
- experiment : req.user_email is a clientId. A use case: clientId is used to deduce the variant.
- feature : req.user_email is a context. A use case : context(an id) is used to deduce if the feature (percentage strategy) is active or not.

BTW, i clean a code !

Reply:
I juste saw you update of documentation.  And you have understood the use of context and clientID !